Documentation/branch.liquid

```diff
@@ -0,0 +1,7 @@
+{% assign url = page.url | split: '/' %}
```

Comment:
Ah, nice to have the include file so we can stop replicating. Note that the rook docs can be generated by the instructions [here](https://github.com/rook/rook.github.io#test). Could you verify with those steps? If not, I'll try to get back to it in the next couple days to verify. Thanks!

Reply:
there is no target name 'run' in the main MakeFile or the one in build/release.

Reply:
> @alexcpn A few final requests, sorry for the delay:
> 
> * There are some merge conflicts. Could you rebase on the latest?
> * Squash the commits and add a more descriptive commit message. The format required by our new bot would be like this:
> 
> ```
> docs: add 'git clone' step to quickstart guides
> 
> Add documentation to clone the correct release branch of the repo
> to reduce confusion from pulling the wrong branch.
> ```

done

Reply:
I could not test this locally; I have tested this Liquid templating in my github acc some time back however